src/components/login/Login.js

```diff
@@ -58,11 +58,28 @@ class Login extends Component {
         this.props.history.push('/create')
     }
 
+    get shouldRenderAccountConfirmationForm() {
+        const { account: { url, accountId } } = this.props
+
+        if (!url) {
+            return undefined
```

Comment:
Just a :nit:, since we're always doing a coercive check on this value currently, but in general we should try to return the same data type for all return branches of a method like this for consistency, or when we do an explicit === check (which is best practice, generally) on the result of this function somewhere else, it would be incorrect just for this one case, or we'd have to check === false || === undefined.

Reply:
Right, I didn't dig into it since I just jumped into this issue to fix it fast - so I refactored the previous exact conditions. I think the best implementation would be to leave `undefined` as it's telling us that we don't know yet if we should show confirmation form or not, and then use it instead of `if (!url)`. Made that change, let me know what do you think?

Reply:
:+1: I can get behind that, the duplication of the !url check in both the render method and the `shouldRenderAccountConfirmationForm` method bothered me a little, actually :D :+1: